rules/configuration.smk

```diff
@@ -396,10 +402,16 @@ if 'replicate' in samples:
     wildcard_constraints:
         sample=any_given('sample', 'replicate'),
         replicate=any_given('replicate'),
+
 if 'condition' in samples:
     wildcard_constraints:
         sample=any_given('sample', 'condition'),
+        assembly=any_given('assembly'),
+        sorting='coordinate|queryname',
+        sorter='samtools|sambamba'
+else:
         condition=any_given('condition'),
```

Comment:
What's happening here?

Firstly, the wildcard constraints are compounding, so assembly was already set. 
Second, your adding condition only if it is NOT in samples?
Third, sorting and sorter are only set of condition in samples? 

I think you want this:

```
if 'condition' in samples:
    wildcard_constraints:
        sample=any_given('sample', 'condition'),
        condition=any_given('condition'),

if 'replicate' in samples and 'condition' in samples:
    wildcard_constraints:
        sample=any_given('sample', 'replicate', 'condition'),

if get_workflow() in ['alignment', 'chip_seq', 'rna_seq', 'ataq_seq']:
    wildcard_constraints:
        sorting='coordinate|queryname',
        sorter='samtools|sambamba',
```

sorter and sorting may even be placed in the default constraints i guess

Reply:
Not sure, I haven't looked at it thoroughly. I guess something in the merge did sth weird. I'll change it.